src/components/MongoWebShell.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SOURCE_URL = 'https://mws.mongodb.com/';
+
+const MongoWebShell = ({
+  nodeData: {
+    options: { version },
+  },
+}) => {
+  return (
+    <iframe
+      className="mws-root"
+      title="MongoDB Web Shell"
+      allowFullScreen
+      sandbox="allow-scripts allow-same-origin"
+      width="100%"
+      height="320"
+      src={version ? SOURCE_URL + `?version=${version}` : SOURCE_URL}
+    ></iframe>
```

Comment:
[nit] We can probably make this iframe self closing:
```suggestion
    />
```